frontend/app/routes/_gcweb-app.personal-information.phone-number.confirm.tsx

```diff
@@ -48,7 +48,7 @@ export async function action({ request }: ActionFunctionArgs) {
   await userService.updateUserInfo(userId, userInfo);
   session.unset('newPhoneNumber');
 
-  return redirectWithSuccess('/personal-information', 'Phone number has been updated.', {
+  return redirectWithSuccess('/personal-information', 'gcweb:toast.phone-number-updated', {
```

Comment:
This new toast i18n key doesn't really belong in the `gcweb` namespace in my opinion since the text isn't part of the GCWeb template. Consider moving the key into the existing `personal-information` namespace (specifically within `phone-number.confirm`).